collector/src/main/java/com/bitmovin/analytics/data/EventData.kt

```diff
@@ -73,4 +73,6 @@ class EventData(bitmovinAnalyticsConfig: BitmovinAnalyticsConfig, val impression
     var audioLanguage: String? = null
     var drmType: String? = null
     var drmLoadTime: Long? = null
+    var videoStartFailed: Boolean? = false
```

Comment:
no default value needed
```suggestion
    var videoStartFailed: Boolean?
```

Reply:
That way there is `Property must be initialized or be abstract` error.